airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorMetadataDefinitionV0.yaml

```diff
@@ -25,6 +25,7 @@ properties:
       - githubIssueLabel
       - connectorSubtype
       - releaseStage
+    additionalProperties: false
```

Comment:
Double checking that the workflow we want to add new metadata properties would be to first update this file, and then add the new properties.  There's not a world in which we would want to allow new-but-as-of-yet unused metadata properties in first?

Reply:
I think if you are adding a new but not yet used you can still add it to the metadata file.

This just prevents these cases
1. You specify `public: false` thinking it will do something
2. You think you set `connectorSubtype` but in reality you set `connectorSubtyp`

Reply:
> You specify public: false thinking it will do something

![image](https://media4.giphy.com/media/fzJCPdONK9pHAcviY5/giphy.gif?cid=ecf05e47zbr45as4esysepilrsg188f6d2gevhsytvej8be5&ep=v1_gifs_search&rid=giphy.gif&ct=g)